### PR TITLE
Fixed yii\db\QueryBuilder to inherit ‘INSERT INTO ... SELECT’ SELECT’s params (fixes #13807)

### LINKED ISSUE #13807: Add INSERT INTO SELECT support
there is an issue regarding this added function.

I tried a query like ***insert into ... select ... from ... where ...***

An error says ***[Microsoft][ODBC Driver 11 for SQL Server]COUNT field incorrect or syntax error***

### What steps will reproduce the problem?
***Yii::$app->db->createCommand()->insert(Class1::tableName(), Class2::find()
->select([ ... ])
->where([ 'id' => 1 ])
)->execute()***

### What is the expected result?
***insert into table1 ( ... ) select ... from table2 where id = 1***

### What do you get instead?
An error says ***[Microsoft][ODBC Driver 11 for SQL Server]COUNT field incorrect or syntax error***

but when i check my SQL logs, it prints the query but it is not yet prepared like
***insert into table1 ( ... ) select ... from table2 where id = :qp0***

### Additional info
I tried removing where condition and it works:

***Yii::$app->db->createCommand()->insert(Class1::tableName(), Class2::find()->select([ ... ]))->execute()***

| Q                | A
| ---------------- | ---
| Yii Version | ***2.0.11.2***
| PHP Version| ***5.6.15***
| Operation System | ***MS Windows 8.1***

### PR DESCRIPTION
**Pull request by sergeymakinen**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | yes
| Tests pass?   | yes
| Fixed issues  | #13807 

This PR fixes `QueryBuilder` did not account params in a subquery when building `INSERT INTO … SELECT` introduced in #9420. I didn’t mention a small BC break in `UPGRADE.md` yet.

### CODE REVIEW

File: framework/db/QueryBuilder.php [hunk 1]
```diff
@@ -206,16 +206,19 @@ public function insert($table, $columns, &$params)
      *
      * @param \yii\db\Query $columns Object, which represents select query
      * @param \yii\db\Schema $schema Schema object to qoute column name
+     * @param array $params the parameters to be bound to the generated SQL statement. These parameters will
+     * be included in the result with the additional parameters generated during the query building process.
      * @return array
+     * @throws InvalidParamException if query's select does not contain only named parameters.
```
Comment (samdark) [inline framework/db/QueryBuilder.php]: only named parameters → named parameters only

**Review by samdark — CHANGES_REQUESTED** [review]
Overall looks OK. Method signature change affect SQL query builders only. That should be mentioned in UPGRADE.

**Review by samdark — APPROVED** [review]

File: framework/db/QueryBuilder.php [hunk 2]
```diff
@@ -204,18 +204,21 @@ public function insert($table, $columns, &$params)
     /**
      * Prepare select-subquery and field names for INSERT INTO ... SELECT SQL statement.
      *
-     * @param \yii\db\Query $columns Object, which represents select query
-     * @param \yii\db\Schema $schema Schema object to qoute column name
+     * @param \yii\db\Query $columns Object, which represents select query.
+     * @param \yii\db\Schema $schema Schema object to qoute column name.
```
Comment (arogachev) [inline framework/db/QueryBuilder.php]: `qoute` - `quote`.

**Review by arogachev — APPROVED** [review]

**Review by SilverFire — APPROVED** [review]

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**